src/main/java/webserver/ResponseHeader.java

```diff
@@ -0,0 +1,24 @@
+package webserver;
+
+import java.util.Map;
+
+public class ResponseHeader extends Header {
+    private static final String STATUS_CODE_KEY = "statusCode";
+    private static final String STATUS_TEXT_KEY = "statusText";
+
+    public ResponseHeader(Map<String, String> attributes) {
+        super(attributes);
+    }
```

Comment:
생성자의 접근 제어자를 protected로 변경할 수 있음
```suggestion
    protected ResponseHeader(Map<String, String> attributes) {
        super(attributes);
    }
```